internal/server/banner

```diff
@@ -0,0 +1,5 @@
+ __     _____
```

Comment:
I would probably place the banner into a different folder perhaps assets/{topic?}